apps/voting/app/src/script.js

```diff
@@ -137,17 +165,39 @@ async function createStore(token, tokenSettings) {
  *                     *
  ***********************/
 
-async function castVote(state, { voteId }) {
-  // Let's just reload the entire vote again,
-  // cause do we really want more than one source of truth with a blockchain?
+async function updateConnectedAccount(state, { account }) {
+  return {
+    ...state,
+    connectedAccount: account,
```

Comment:
Let's keep this outside of the cache, by moving this state to be outside of `createStore()`